api/types/v1/types.proto

```diff
@@ -30,6 +30,12 @@ message Point {
   double value = 1;
   // Milliseconds unix timestamp
   int64 timestamp = 2;
+  repeated ProfileAnnotation annotations = 3;
+}
+
+message ProfileAnnotation {
```

Comment:
I think it is also worth documenting, how annotations are different from labels.

- They can't be used at query time to select on.
- There can be multiple annotations with the same key (with labels we don't allow duplicates).